@xen-orchestra/lite/src/components/form/FormInputWrapper.vue

```diff
@@ -55,8 +62,11 @@ const props = withDefaults(
     icon?: IconDefinition;
     learnMoreUrl?: string;
     warning?: string;
+    htmlWarning?: string;
     error?: string;
+    htmlError?: string;
     help?: string;
+    htmlHelp?: string;
```

Comment:
I think it would be preferable to use slots for that.

Something like this:

```vue
<div v-if="warning || $slots.warning" class="warning">
  <slot name="warning">{{ warning }}</slot>
</div>
```

This way, we can pass the message as prop, like now, but if we need HTML we can use a `<template #warning>...</template>` instead.